Source/WebCore/dom/StaticRange.cpp

```diff
@@ -83,4 +83,23 @@ void StaticRange::visitNodesConcurrently(JSC::AbstractSlotVisitor& visitor) cons
     addWebCoreOpaqueRoot(visitor, end.container.get());
 }
 
+bool StaticRange::isValid() const
```

Comment:
Please add a link to https://dom.spec.whatwg.org/#staticrange-valid as a comment.